dbms/src/Storages/KVStore/Decode/DecodingStorageSchemaSnapshot.cpp

```diff
@@ -17,6 +17,95 @@
 
 namespace DB
 {
+DecodingStorageSchemaSnapshot::DecodingStorageSchemaSnapshot(
```

Comment:
Make these as comments in the code rather than a PR comment in the page?
![image](https://github.com/pingcap/tiflash/assets/4865550/ecb3e388-a41c-4ec8-be20-437a5e1fee35)

Reply:
I'm going to remove them from the page. It takes the space on webpage and make some trouble to review the logic.

Reply:
OK